build_runner/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## 0.8.4
 
 - Log the number of completed actions on successful builds.
+- Deprecated `PostProcessBuilder` and `PosProcessBuildStep`. These should be
```

Comment:
I wouldn't say deprecated. They were completely removed...

Reply:
They are still `export`ed so that it's not a breaking change, but using them through a build_runner imports is what is deprecated.